src/resolvers/Query/organizations.ts

```
@@ -1,53 +1,33 @@
-import type { QueryResolvers } from "../../types/generatedGraphQLTypes";
+/**
+ * Resolver for querying organizations.
+ * @returns Array of organization objects
+ */
+import { QueryResolvers } from "../../types/generatedGraphQLTypes";
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)